### PR TITLE
refactor: [M3-7437] - Query Key Factory for Security Questions and Preferences

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Moves profile related queries into `src/queries/profile` folder 📁 because profile-related queries are spread across many files
  - This caused the massive diff, sorry 😣
- Uses query key factory for security questions 🏭 
- Uses query key factory for preferences 🏭   

## How to test 🧪

- Verify the profile loads
- Verify security questions work (on http://localhost:3000/profile/auth) (covered by existing cypress test)
- Verify preferences work (toggling the primary nav uses preferences)

## As an Author I have considered 🤔

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/queries/profile/securityQuestions.ts
Comment (bnussman-akamai) [inline packages/manager/src/queries/profile/securityQuestions.ts:1]: This isn't new code, it is just the old code moved into a new file with the query key factory used

File: packages/manager/src/queries/profile/profile.ts [hunk 1]
```diff
@@ -55,10 +57,18 @@ export const profileQueries = createQueryKeys('profile', {
     queryFn: () => getPersonalAccessTokens(params, filter),
     queryKey: [params, filter],
   }),
+  prefernces: {
```
Comment (abailly-akamai) [inline packages/manager/src/queries/profile/profile.ts:60]: You've got a typo here that'll need to be updated through the PR

**Review by abailly-akamai — APPROVED** [review]
Minus the typo, I haven't found any regression or issue with the PR. e2e suite looks healthy too. Thanks for the cleanup!

**Review by jaalah-akamai — APPROVED** [review]
✅ Checked Parent/Child accounts too and these look good

**Pull request merged**